### PR TITLE
update code example to include timeout parameter

### PR DESCRIPTION
**Pull request by mdcruz**
Address feedback from https://k6io.slack.com/archives/C02LAJA6EJ1/p1679082666447019

### CODE REVIEW

**Review by ankur22 — APPROVED** [review]
LGTM

File: src/data/markdown/docs/30 k6-browser/01 Get started/02 Running browser tests.md [hunk 1]
```diff
@@ -43,7 +46,9 @@ To run a simple local script:
...
+  The preceding code imports the `chromium` [BrowserType](/javascript-api/k6-browser/api/browsertype) (currently the only available `BrowserType` implementation), and uses its `launch` method to start up a Chromium [Browser](/javascript-api/k6-browser/api/browser) process. Two parameters are passed to it. One is the `headless` parameter with the value `false` so you can see the browser launching, and `timeout` parameter with the value `60s` which will be the timeout use for various actions and navigation. For a full list of parameters that you can pass, check out the documentation for [BrowserType.launch()](/javascript-api/k6-browser/api/browsertype/launch/).
```
[the remark this answers is not in the document]
Comment (ankur22) [reply src/data/markdown/docs/30 k6-browser/01 Get started/02 Running browser tests.md:49]: I missed that. You mean:

```suggestion
  The preceding code imports the `chromium` [BrowserType](/javascript-api/k6-browser/api/browsertype) (currently the only available `BrowserType` implementation), and uses its `launch` method to start up a Chromium [Browser](/javascript-api/k6-browser/api/browser) process. Two parameters are passed to it. One is the `headless` parameter with the value `false` so you can see the browser launching, and `timeout` parameter with the value `60s` which will be the timeout used for various actions and navigation. For a full list of parameters that you can pass, check out the documentation for [BrowserType.launch()](/javascript-api/k6-browser/api/browsertype/launch/).
```

[the remark this answers is not in the document]
Comment (ka3de) [reply src/data/markdown/docs/30 k6-browser/01 Get started/02 Running browser tests.md:49]: Yes, that sounds better to me.

**Review by inancgumus — APPROVED** [review]

File: src/data/markdown/docs/30 k6-browser/01 Get started/02 Running browser tests.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mdcruz) [reply src/data/markdown/docs/30 k6-browser/01 Get started/02 Running browser tests.md]: Thanks, both! I have committed the suggestion.

**Pull request merged**